Skaffold API docs (#3068)
GoogleContainerTools/skaffold: docs/content/en/docs/references/api/_index.md

<a name="proto.BuildEvent"></a>

### BuildEvent

> BuildEvent describes if the build status per artifact. Status could be one of

Reviewer: ```suggestion
BuildEvent describes the build status per artifact, and will be emitted by Skaffold anytime a build starts or finishes, successfully or not. If the build fails, an error will be attached to the event.
```

> &#34;InProgress&#34;, &#34;Completed&#34; or &#34;Failed&#34;.

Reviewer: ```suggestion
```

<a name="proto.DeployEvent"></a>

### DeployEvent

> Deploy Event describes if the deployment has started, is in progress or is complete.

Reviewer: ```suggestion
DeployEvent gives the status of a deployment, and will be emitted by Skaffold anytime a deployment starts or completes, successfully or not.
If the deployment fails, an error will be attached to the event.
```
